src/data/schemas/customers/customer.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { COUNTRY } from "data/salesPortal/customers/country";
+
+export const CustomerSchema = {
```

Comment:
[Rules by identifier type](https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type)
```suggestion
export const customerSchema = { // CUSTOMER_SCHEMA
```